src/commands/RunCommand.js

```diff
@@ -42,8 +42,7 @@ export default class RunCommand extends Command {
   }
 
   runScriptInPackage(pkg, callback) {
-    NpmUtilities.runScriptInDir(this.script, this.args, pkg.location, (err, stdout) => {
-      this.logger.info(stdout);
+    NpmUtilities.runScriptInDir(this.script, this.args, pkg.location, (err) => {
       if (err) {
```

Comment:
if i leave that in, it's logs stdout twice, since `runScriptInDir` uses `exec`.

is that what you're asking?